packages/swapper/src/swappers/utils/helpers/helpers.ts

```diff
@@ -94,7 +95,7 @@ export const grantAllowance = async ({
   adapter,
   erc20Abi,
   web3
-}: GrantAllowanceArgs): Promise<string> => {
+}: GrantAllowanceArgs<ZrxSupportedChainIds>): Promise<string> => {
```

Comment:
Seeing as how this is being used in all eth swappers (cow, zrx, thor), we should maybe have a concept of `EVMSupportedChainIds` that we can use where suitable

Reply:
That checks out - I've created an `EvmSupportedChainIds` type and replaced `ZrxSupportedChainIds` with it in https://github.com/shapeshift/lib/pull/850/commits/c336ab95ee3fd0b00590e8cb2bccd1a0f6df9705